pages/_app.tsx

```diff
@@ -157,6 +177,8 @@ export default function App({ Component, pageProps }: AppProps) {
           const balance = await state?.connection.tz.getBalance(userAddress);
           dispatch({
             type: "login",
+            // TODO: FIX
```

Comment:
Both are here because the version of beacon and taquito's beacon are not the same, so there's a type error. Hopefully we can just ignore it and it works, but we'll need to remove the `@ts-ignore` once taquito will updates